docs/guides/mlp_tutorials/llm-finetuning.md

```diff
@@ -0,0 +1,170 @@
+[](){#ref-mlp-llm-finetuning-tutorial}
+
+# LLM Finetuning Tutorial
```

Comment:
Have a look at https://eth-cscs.github.io/cscs-docs/contributing/#style-guide for some general guidelines on styling/formatting. We're not aiming for perfect, nothing is blocking, but we try to move towards some consistency.